lib/nearby.js

```diff
@@ -18,16 +19,17 @@ fns.getLocationsNearby = (requestData, done) => {
     Joi.validate(requestData.data, validation.locationsNearby, (err, data) => {
 
         if (err) {
-            return done(err);
+            return util.validationError(err, 'location get service', done);
         }
+
         db.findDataNearby('locations', data.long, data.lat, {
-                maxDistance: data.maxDistance / 6371,
+                maxDistance: data.maxDistance / 6371, //TODO: what? why 6371? what if undefined?
```

Comment:
as you can see in https://github.com/locator-kn/ms-location/blob/get-locations-nearby/lib/nearby.js#L14 maxDistance will never be `undefined`